doc/source/examples/unresolved-cfd-dem/boycott-effect/boycott-effect.rst

```diff
@@ -0,0 +1,372 @@
+==================================
+Boycott Effect
+==================================
+
+It is strongly recommended to visit `DEM parameters <../../../parameters/dem/dem.html>`_  and `CFD-DEM parameters <../../../parameters/unresolved-cfd-dem/unresolved-cfd-dem.html>`_ for more detailed information on the concepts and physical meaning of the parameters ind DEM and CFD-DEM.
+
+Features
+----------------------------------
+- Solvers: ``dem_3d`` and ``cfd_dem_coupling_3d``
+- Three-dimensional problem
+- Displays the selection of models and physical properties.
+- Simulates a solid-liquid sedimentation.
+
+
+Files used in this example
+---------------------------
+``/examples/unresolved-cfd-dem/boycott-effect``
+
+
+Description of the case
+-----------------------
+
+This example simulates the sedimentation of a group of particles in a viscous fluid. Two cases where simulated. In the first case, the tube was vertically placed. In the second case, the tube was inclined at a :math:`20^{\circ}` with the vertical. First, we use Lethe-DEM to insert the particles. We enable check-pointing in order to write the DEM checkpoint files which will be used as the starting point of the CFD-DEM simulation. Then, we use the ``cfd_dem_coupling_3d`` solver within Lethe to simulate the sedimentation of particles by initially reading the checkpoint files from the DEM simulation.
```

Comment:
In my opinion, it would be nice to have an image of the channels with particles inserted in this DEM section.